client/test/cypress/plugins/index.js

```diff
@@ -36,4 +36,15 @@ module.exports = (on, config) => {
       });
     }
   });
+
+  on('before:browser:launch' , (browser = {}, launchOptions) => {
+    const REDUCE = 1;
+    if (browser.family === 'firefox') {
+      launchOptions.preferences['ui.prefersReducedMotion'] = REDUCE;
```

Comment:
I'm curious about the choice to use a local variable here.

Reply:
It was in the example code. I just kept it. 🤷‍♀️

I thought the same thing.